action/transfer.go

```diff
@@ -77,12 +61,12 @@ func (tsf *Transfer) Destination() string { return tsf.recipient }
 
 // TotalSize returns the total size of this Transfer
 func (tsf *Transfer) TotalSize() uint32 {
-	size := tsf.BasicActionSize()
+	size := uint32(len(tsf.payload))
```

Comment:
BasicActionSize() is not 0, TotalSize() has changed as a result.
if TotalSize() is used in calculating gas charge, might be an issue